source/getting-started.txt

```diff
@@ -0,0 +1,134 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Tutorials </tutorials>
+
+The resources on this page are designed to help you explore Atlas App 
+Services. To explore a working codebase that showcases App Services' 
+features, check out a Template App. For a more guided experience in
+learning how to develop with App Services and Realm, start with a 
+Tutorial.
+
+
+Template Apps
+-------------
+
+Bootstrap App Services with a Template App. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. The following template
+apps are available:
+
+- A mobile todo app written with Realm SDKs that syncs to an App Services App
+- A hosted todo web app using the GraphQL API or Realm Web SDK
+- An Event-driven :doc:`Database Trigger </triggers/database-triggers>` 
+  template that updates a view in a separate collection.
+
+Template apps are available for the following languages:
+
+- C# (Xamarin)
+- Dart (Flutter)
+- JavaScript (React Native)
+- Kotlin (Android)
+- Swift (iOS + SwiftUI)
```

Comment:
rather than listing them all here, give a couple of examples (say 3), and then link out to the template app page that's now in the reference section.